sim/shaman/chain_lightning.go

```diff
@@ -51,7 +42,7 @@ func (shaman *Shaman) newChainLightningSpell(isElementalOverload bool) *core.Spe
 			totalDamage := baseDamage * bounceReduction
 			result := spell.CalcDamage(sim, curTarget, totalDamage, spell.OutcomeMagicHitAndCrit)
 
-			if canOverload && result.Landed() && sim.RandomFloat("Chain Lightning Elemental Overload") <= overloadChance/3 {
+			if result.Landed() && sim.RandomFloat("Chain Lightning Elemental Overload") <= shaman.GetOverloadChance()/3 {
```

Comment:
`sim.Proc(chance, label)` is the preferred method to use instead of manual `sim.RandomFloat`